common/src/main/python/dlpx/virtualization/common/VERSION

```diff
@@ -1 +1 @@
-2.0.0-internal-001
\ No newline at end of file
+2.0.0-internal-2
```

Comment:
002?

Reply:
The version is automatically bumped when a developer runs `bumpversion build`. I talked to Filip about this offline, and we agreed the leading zeroes are not important and that we should keep the build version bumping automated.

Reply:
Sounds good.